src/Sulu/Bundle/MediaBundle/DependencyInjection/Configuration.php

```diff
@@ -62,12 +62,26 @@ public function getConfigTreeBuilder()
                     ->arrayNode('default_imagine_options')
                         ->prototype('scalar')->end()
                     ->end()
+                    ->arrayNode('mime_types')
+                        ->prototype('scalar')->end()->defaultValue(array(
+                                                                'image/jpeg',
+                                                                'image/jpg',
+                                                                'image/gif',
+                                                                'image/png',
+                                                                'image/bmp',
+                                                                'image/svg+xml',
+                                                                'image/vnd.adobe.photoshop',
+                                                                'application/pdf',
+                                                            ))
+                    ->end()
                 ->end()
             ->end()
             ->arrayNode('format_cache')
                 ->addDefaultsIfNotSet()
                 ->children()
-                    ->scalarNode('public_folder')->defaultValue(SuluMediaExtension::DEFAULT_FORMAT_CACHE_PUBLIC_FOLDER)->end()
+                    ->scalarNode('path')->defaultValue('%assetic.write_to%/uploads/media')->end()
```

Comment:
@alexander-schranz Sry, didn't read the second paragraph... Yeah, I would go there over all bundles. It's in a compiler pass, and therefore cached anyway. And to me it is really important that we don't force the people to use assetic, since my feeling from several conferences is that more and more people ditch assetic in favour of grunt/gulp.

And would my previous comment work for you?